apps/api/infra/api.ts

```diff
@@ -132,6 +132,16 @@ export const serviceSetup = (services: {
       PARTY_LETTER_REGISTRY_BASE_API_URL: ref(
         (h) => `http://${h.svc(services.servicesPartyLetterRegistryApi)}`,
       ),
+      DRIVING_LICENSE_PATH_REPLACEMENT_FROM: {
+        prod: '/RafraentOkuskirteini-v1/api/Okuskirteini/',
+        staging: '',
+        dev: '',
+      },
+      DRIVING_LICENSE_PATH_REPLACEMENT_TO: {
+        prod: '/RafraentOkuskirteini-v1/api/okuskirteini/',
+        staging: '',
+        dev: '',
+      },
```

Comment:
Really? 🙈

Can they not fix this on their end?

Reply:
they can, and they have it planned.. but yes, really (for now)